pkgs/applications/video/mpv/scripts/webtorrent-mpv-hook.nix

```diff
@@ -21,6 +21,10 @@ buildNpmPackage rec {
   npmDepsHash = "sha256-GpNUJ5ZCgMjSYLqsIE/RwkTSFT3uAhxrHPe7XvGDRHE=";
   makeCacheWritable = true;
 
+  nativeBuildInputs = [
+    python3 # Fixes node-gyp on aarch64-linux
```

Comment:
very weird that its only required on aarch64-linux, please add a comment with the error